docs/advanced.rst

```diff
@@ -46,7 +63,6 @@ Another example is related to the `feature request #143
 at the beginning and the end of a file.  This use case is easy to implement
 through a custom wrapper for the PEP 8 library::
 
-  #!python
```

Comment:
It's meant to indicate that this is an executable script. Regardless, the purpose of your PR was to *add* examples, not change existing ones.

```
#!
```

At the top of a file indicates that the script can be executed by the thing after `#!`, e.g., 

```
#!python
#!/usr/bin/env python
#!/usr/bin/env bash
#!/usr/bin/env perl
```